### DISCUSSION TITLE
Mobile theme

### DISCUSSION THREAD

**Discussion by antcorro** [YetiForceCompany/YetiForceCRM#14881] [How to] [answered]
Hello, here [https://github.com/YetiForceCompany/YetiForceCRM/issues/1564](#1564) I see that "In YetiForce the mobile version, that has always been available in /modules/Mobile, was disabled by default [there were some security errors that we haven't dealt with yet]. You can enable it here [at your own risk] : https://github.com/YetiForceCompany/YetiForceCRM/blob/stable/config/api.php.". This is an old issue and now the path blob/stable/config/api.php does not exist
How can I enable the mobile version?

YetiForceCRM v6.1.0

Thanks
Antonio

**Comment by bpabiszczak** [comment, marked as answer]
This functionality no longer exists.

**Reply by antcorro** [reply]
ok, thank you.
So there is no way to use Yeti Force via Mobile device?

**Reply by bpabiszczak** [reply]
The system is responsive, run it on your phone. Additionally, you have IMAP, CalDAV and CardDAV integration in the system.
